authors/apps/authentication/messages.py

```diff
@@ -16,6 +16,8 @@
     "missmatch":"Email and password did not match",
     "require_pwd":"Password is required to login",
     "email_pwd":"Email address is required to login",
+    "pfl_err":"User profile not found",
```

Comment:
@Ruiru11 this is also fine but it would be better if you could put a more descriptive keyword. The whole point is for this is re-usability

Reply:
worked on that